src/config/chains/telos-evm-testnet/index.ts

```diff
@@ -1,79 +1,221 @@
-    network: NETWORK,
-    decimals: 18,
-    address: NativeCurrencyAddress,
-    logo: LOGO,
-    logoURI: LOGO,
-    isNative: false,
-    isSystem: true,
-} as TokenSourceInfo);
-
-const S_TOKEN = new TokenClass({
-    name: 'Staked Telos',
-    symbol: 'STLOS',
-    network: NETWORK,
-    decimals: 18,
-    address: '0xa9991E4daA44922D00a78B6D986cDf628d46C4DD',
-    logo: 'https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/stlos.png',
-    isNative: false,
-    isSystem: false,
-} as TokenSourceInfo);
-
-const W_TOKEN = new TokenClass({
-    name: 'Wrapped Telos',
-    symbol: 'WTLOS',
-    network: NETWORK,
-    decimals: 18,
-    address: '0xaE85Bf723A9e74d6c663dd226996AC1b8d075AA9',
-    logo: 'https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/wtlos.png',
-    isNative: false,
-    isSystem: false,
-} as TokenSourceInfo);
-
-const RPC_ENDPOINT = {
-    protocol: 'https',
-    host: 'testnet.telos.net',
-    port: 443,
-    path: '/evm',
+const config: NetworkConfig =
+{
+    'network': 'telos-evm-testnet',
+    'display': 'Telos EVM (Testnet)',
+    'title': 'TelosEVM Testnet Explorer',
+    'chainId': '41',
+    'isTestnet': true,
+    'logos': {
+        'large': 'https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/telos.png',
+        'small': 'https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/telos.png',
+    },
+    'systemTokens': [
+        {
+            'name': 'Telos',
+            'symbol': 'TLOS',
+            'network': 'telos-evm-testnet',
+            'decimals': 18,
+            'address': NativeCurrencyAddress,
+            'logo': 'https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/telos.png',
+            'isNative': false,
+            'isSystem': true,
+        },
+        {
+            'name': 'Staked Telos',
+            'symbol': 'STLOS',
+            'network': 'telos-evm-testnet',
+            'decimals': 18,
+            'address': '0xa9991E4daA44922D00a78B6D986cDf628d46C4DD',
+            'logo': 'https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/stlos.png',
+            'isNative': false,
+            'isSystem': false,
+        },
+        {
+            'name': 'Wrapped Telos',
+            'symbol': 'WTLOS',
+            'network': 'telos-evm-testnet',
+            'decimals': 18,
+            'address': '0xaE85Bf723A9e74d6c663dd226996AC1b8d075AA9',
+            'logo': 'https://raw.githubusercontent.com/telosnetwork/token-list/main/logos/wtlos.png',
+            'isNative': false,
+            'isSystem': false,
+        },
+    ],
+    'escrowContractAddress': '0x7E9cF9fBc881652B05BB8F26298fFAB538163b6f',
+    'apiEndpoints': {
+        'rpc': {
+            'protocol': 'https',
+            'host': 'rpc.testnet.telos.net',
+            'port': 443,
+            'path': '/evm',
+        },
+        'api': 'https://api-dev.telos.net/v1',
+        'hyperion': 'https://rpc.testnet.telos.net',
```

Comment:
@zkjamesfish provided those new links in this PR he created:
https://github.com/telosnetwork/teloscan/pull/909